digitalocean/Firewall.py

```diff
@@ -195,6 +195,22 @@ def create(self, *args, **kwargs):
 
         return self
 
+    def update(self, *args, **kwargs):
+        inbound = jsonpickle.encode(self.inbound_rules, unpicklable=False)
+        outbound = jsonpickle.encode(self.outbound_rules, unpicklable=False)
+        params = {'name': self.name,
```

Comment:
Code is repeated, think about using a function to get the params or refactor to avoid repeated code